<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FTP_serverOptions_Dialog</class>
 <widget class="QDialog" name="FTP_serverOptions_Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>312</width>
    <height>196</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Server Options</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>291</width>
     <height>81</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Server Port</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="port_lineEdit">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>10</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>1234</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>52</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Home Dir</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="homeDir_lineEdit">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>50</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="homeDirBrowse_pushButton">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>50</y>
      <width>31</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="btr_frame">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>291</width>
     <height>80</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Max Bitrate</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="bitrate_lineEdit">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>50</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QCheckBox" name="usebitrate_checkBox">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>10</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Use realtime data transfer</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
